### DISCUSSION TITLE
benefits of stylix compare to others

### DISCUSSION THREAD

**Discussion by luxus** [nix-community/stylix#10] [General]
Hello,
I'm not such a nix pro yet, but I wonder what are the benefits of stylix compared to https://github.com/Misterio77/nix-colors

**Comment by danth** [comment]
You could describe `nix-colors` as setting a constant, which you have to refer to in your own configuration for it to have an effect.

Stylix goes a step further using pre-made configuration, so your colour scheme is used in supported apps without needing any extra work on your behalf

**Comment by luxus** [comment]
thanks, i will will try it next to each other. 
i think its easier to implement your approach to lunarvim (a nvim with a lot plugins) as a theme .
and from my understanding it should be pretty easy to move from nix-colors implimentation to stylix 1 by 1 because it supports settings constants as well right?

**Reply by danth** [reply]
Yeah. Stylix exports the colors as `config.lib.stylix.colors.base00` to `config.lib.stylix.colors.base0F`, so you should be able to move from `nix-colors` pretty easily.
